src/dashboard/librarysettings.js

```diff
@@ -2,45 +2,47 @@ define(["jQuery", "loading", "libraryMenu", "fnchecked", "emby-checkbox", "emby-
     "use strict";
 
     function loadPage(page, config) {
-        config.MergeMetadataAndImagesByName ? $(".fldImagesByName", page).hide() : $(".fldImagesByName", page).show(), $("#chkSaveMetadataHidden", page).checked(config.SaveMetadataHidden), $("#txtMetadataPath", page).val(config.MetadataPath || ""), $("#txtMetadataNetworkPath", page).val(config.MetadataNetworkPath || ""), loading.hide()
```

Comment:
why check for `config.MergeMetadataAndImagesByName` is gone?

Reply:
It was one of the dead settings, there was no UI for this on the page.